components/board/status.go

```diff
@@ -20,7 +20,7 @@ func CreateStatus(ctx context.Context, b Board) (*pb.Status, error) {
 			if err != nil {
 				return nil, err
 			}
-			val, err := x.Read(ctx, nil)
+			val, _, err := x.Read(ctx, nil)
```

Comment:
Discussion question: the status returned here contains a map from analog readers to their values. Should that be modified to also include data on the values' range? Part of me says "the current implementation is right: people care about the values, not the range." and part of me says "the point of this PR is that people care about the range, too, and it should be attached to any values we provide."

Reply:
Yeah we can change the status here but I think this will affect the frontend RC card stuff so I will change this in another PR

Reply:
I'm not even sure it _should_ be changed; just that it's worth discussing. As you say, this will affect the RC card and maybe we shouldn't change that.